plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -323,7 +327,11 @@ createWidget("discourse-poll-container", {
 
     if (attrs.showResults) {
       const type = poll.get("type") === "number" ? "number" : "standard";
-      return this.attach(`discourse-poll-${type}-results`, attrs);
+      const resultsWidget =
+        type === "number" || attrs.poll.chart_type !== "pie"
```

Comment:
These should use the same `const` as the other file. You can `export const` and then import them here to compare.